install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -59,6 +59,9 @@ spec:
 {{- else }}
           - --insecure=true
 {{- end }}
+{{- if $.Values.global.enableServiceDiscovery }}
```

Comment:
Where is this option? I think it needs to be in values.yaml so it is documented.

Why is it global and not galley? I am not sure what it means to globally enableServiceDiscovery.. isn't that what pilot is?

Reply:
@howardjohn updated